collated/main/A0140022H.md

```diff
@@ -0,0 +1,647 @@
+# A0140022H
+###### \java\tars\commons\util\DateTimeUtil.java
+``` java
+     */
+    public static String modifyDate(String dateToModify, String frequency) {
+        LocalDateTime date = LocalDateTime.parse(dateToModify, formatter);
+
+        switch (frequency.toLowerCase()) {
+        case DATETIME_DAY:      date = date.plusDays(DATETIME_INCREMENT);
+                                break;
+        case DATETIME_WEEK:     date = date.plusWeeks(DATETIME_INCREMENT);
+                                break;
+        case DATETIME_MONTH:    date = date.plusMonths(DATETIME_INCREMENT);
+                                break;
+        case DATETIME_YEAR:     date = date.plusYears(DATETIME_INCREMENT);
+                                break;
+        }
+
+        dateToModify = date.format(stringFormatter);
+        return dateToModify;
+    }
+    
+    public static LocalDateTime setLocalTime(LocalDateTime dateTime, int hour, int min, int sec) {
+        return LocalDateTime.of(dateTime.getYear(), dateTime.getMonth(), dateTime.getDayOfMonth(),
+                hour, min, sec);
+    }
+}
+```
+###### \java\tars\commons\util\ExtractorUtil.java
+``` java
+     */
+    public static String[] getRecurringFromArgs(String recurringArguments, Prefix prefix)
+            throws IllegalValueException {
+        recurringArguments = recurringArguments.replaceFirst(prefix.prefix, EMPTY_STRING).trim();
+        String[] recurringString = recurringArguments.split(ONE_EMPTY_SPACE);
+
+        return recurringString;
+    }
+}
+```
+###### \java\tars\logic\commands\AddCommand.java
+``` java
+ */
+public class AddCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a task to tars. "
+            + "Parameters: NAME [/dt DATETIME] [/p PRIORITY] [/t TAG] [/r NUM_TIMES FREQUENCY]...\n " + "Example: "
```

Comment:
Avoid lines longer than 80 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer.